data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -1001,24 +1053,28 @@
     "    E.g., it might show that tracts prioritized by A but not B have a higher average income,\n",
     "    or that tracts prioritized by B but not A have a lower percent of unemployed people.\n",
     "    \"\"\"\n",
-    "    df_subset = df[\n",
-    "        [\n",
-    "            method_a_priority_census_tracts_field,\n",
-    "            method_b_priority_census_tracts_field,\n",
-    "        ]\n",
-    "        + comparison_fields\n",
+    "    fields_to_group_by = [\n",
+    "        method_a_priority_census_tracts_field,\n",
+    "        method_b_priority_census_tracts_field,\n",
     "    ]\n",
     "\n",
+    "    df_subset = df[fields_to_group_by + comparison_fields]\n",
+    "\n",
     "    grouped_df = df_subset.groupby(\n",
-    "        [\n",
-    "            method_a_priority_census_tracts_field,\n",
-    "            method_b_priority_census_tracts_field,\n",
-    "        ],\n",
+    "        fields_to_group_by,\n",
     "        dropna=False,\n",
     "    )\n",
     "\n",
-    "    # Run the comparison function on the groups.\n",
-    "    comparison_df = grouped_df.mean().reset_index()\n",
+    "    # Take the mean of all fields.\n",
+    "    comparison_df = grouped_df.mean()\n",
+    "\n",
+    "    # Also add in the count of census tracts.\n",
```

Comment:
All these updates in this section are to add in the count of census tracts per groupby.